Library/Formula/duck.rb

```diff
@@ -2,15 +2,12 @@ class Duck < Formula
   desc "Command-line interface for Cyberduck (a multi-protocol file transfer tool)"
   homepage "https://duck.sh/"
   # check the changelog for the latest stable version: https://cyberduck.io/changelog/
-  url "https://dist.duck.sh/duck-src-4.7.3.18396.tar.gz"
-  sha256 "47e25f0a28393c388f37d319c9d51dd51eebdf15198ee48df3995af8a60bcc16"
+  url "https://dist.duck.sh/duck-src-4.8.18316.tar.gz"
+  sha256 "d48c209a65b587143fc3b8e98290edbe6e354c6ea710c095507e5be7ddb0f048"
   head "https://svn.cyberduck.io/trunk/"
 
   bottle do
     cellar :any
-    sha256 "357282a2d6092927a8dd133e9868646dffab55b339376280b8476d19cf7db6b8" => :el_capitan
```

Comment:
Restored bottle block to original, still same problem. 
Note, I edit file from github web interface if that makes any diff with UTF-8 error.

thanks